test/testcases/test_web_api/test_canvas_app/test_switch_component_unit.py

```
@@ -0,0 +1,380 @@
+    ``Graph.get_variable_param_value`` used by the canvas resolver."""
+    cur = obj
+    for key in path.split("."):
+        if cur is None:
+            return None
+        if isinstance(cur, dict):
+            cur = cur.get(key)
+            continue
+        if isinstance(cur, (list, tuple)):
+            try:
+                cur = cur[int(key)]
+            except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)